internal/provider/resource_user.go

```diff
@@ -43,6 +43,11 @@ func resourceUser() *schema.Resource {
 				Type:        schema.TypeString,
 				Required:    true,
 			},
+			"active": {
+				Description: "Active",
+				Type:        schema.TypeBool,
+				Required:    false,
```

Comment:
I think there needs to be a `Default` attribute set here. https://www.terraform.io/docs/extend/schemas/schema-behaviors.html#default

Reply:
sorry about that and thanks for your help, I'm not familiar with Go or Terraform modules either